### PR TITLE
Updating wof:name, name:eng_x_preferred, and {lang}_x_preferred for non-locality placetypes

### PR DESCRIPTION
**Pull request by ndcartography**
**Objective:** Fixes https://github.com/mapzen/data-tasks/issues/32 and https://github.com/whosonfirst-data/whosonfirst-data/issues/843.

- Copy any English `wof:name` into `name:eng_x_preferred`
- Update any non-english `wof:name` records to English (and store non-English as preferred language name)
    - **Example:** `Río de la Plata` would go into Spanish preferred name, `wof:name` would be updated to `Plata River`, and preferred English name would go to `Plata River`.

**Placetypes covered:**

- [x] dependency
- [x] ocean
- [x] continent
- [x] country
- [x] marine area
    - [x]  (will be taken care of when we fix merge conflicts) For features mod'd in https://github.com/whosonfirst-data/whosonfirst-data/issues/843, go back and make sure their non-English name is restored... and the default wof:name is also set as English preferred name.
- [x] macroregion
- [x] **region** _easy ones only, let GeoNames alt name import take care of most the rest_

### CODE REVIEW

**Comment by ndcartography** [comment]
Fixes issue https://github.com/mapzen/data-tasks/issues/32

**Comment by nvkelso** [comment]
Also references statistics generated as part of https://github.com/mapzen/data-tasks/issues/31.

English %

**Comment by ndcartography** [comment]
NOTE: Final percent coverage chart for `eng_x_preferred` should be produced after all commits have been made.

File: data/404/528/741/404528741.geojson [hunk 1]
```diff
@@ -60,6 +61,9 @@
     "name:ell_x_preferred":[
         "\u0398\u03ac\u03bb\u03b1\u03c3\u03c3\u03b1 \u039c\u03c0\u03bf\u03c6\u03cc\u03c1"
     ],
+    "name:eng_x_preferred":[
+        "Beaufort  Sea"
```
Comment (nvkelso) [inline data/404/528/741/404528741.geojson]: Remove extra space here and in wof:name.

File: data/404/528/743/404528743.geojson [hunk 2]
```diff
@@ -114,6 +115,9 @@
     "name:ell_x_preferred":[
         "\u039a\u03b1\u03c1\u03b1\u03ca\u03b2\u03b9\u03ba\u03ae \u0398\u03ac\u03bb\u03b1\u03c3\u03c3\u03b1"
     ],
+    "name:eng_x_preferred":[
+        "Caribbean  Sea"
```
Comment (nvkelso) [inline data/404/528/743/404528743.geojson]: Extra space, again.

File: data/404/528/745/404528745.geojson [hunk 3]
```diff
@@ -5,13 +5,17 @@
...
+    "geom:area_square_m":1404742276784.713379,
     "geom:bbox":"-97.8575872465,20.7999968531,-79.8955570137,30.694354153",
     "geom:latitude":25.490732,
     "geom:longitude":-89.914791,
     "iso:country":"",
     "lbl:latitude":25.255128,
     "lbl:longitude":-90.053644,
     "mz:hierarchy_label":1,
+    "name:eng_x_preferred":[
+        "Gulf Of Mexico"
```
Comment (nvkelso) [inline data/404/528/745/404528745.geojson]: Of > of

Comment (ndcartography) [inline data/404/528/745/404528745.geojson]: noted ^^^

**Comment by ndcartography** [comment]
Before and after status report for name:eng_x_preferred records from this PR: 

eng_percent

eng_percent

_regions are estimated to be at 100% after geonames concordance work is completed_

**Comment by stepps00** [comment]
Overall, this looks good.

Noting that this PR does a bit more than just update `name:eng_x_preferred` properties.

I'm seeing edits to the following properties:
- `wof:name`
- `name:eng_x_preferred`
- `name:[lang]_x_preferred`

Are there additional properties updated in this PR?

**Comment by ndcartography** [comment]
@stepps00 Correct; for some commits, the `wof:name` is updated first, then a second commit moves the updated `wof:name` into `eng_x_preferred` 

This way, bunk `wof:name` records don't contaminate the `eng_x_preferred`

**Comment by nvkelso** [comment]
Over to @thisisaaronland, this is property only changes that don't need PIPing.

**Comment by thisisaaronland** [comment]
Can someone please take a look at these merge conflicts and then reassign back to me.

**Comment by ndcartography** [comment]
@thisisaaronland Working on it now. I'm not sure where these ones came from, since we just fixed all merge conflicts earlier...

**Comment by thisisaaronland** [comment]
We're merging things all over the place right now, so it's not really surprising.

**Comment by stepps00** [comment]
@ndcartography - there were some recent (today) merges that added additional conflicts.

**Comment by stepps00** [comment]
Note: this PR is complete, will be a top priority for merge once conflicts are taken care of.

**Comment by ndcartography** [comment]
@stepps00 @nvkelso @thisisaaronland - as of right now, there are now all merge conflicts have been dealt with.

**Comment by nvkelso** [comment]
Over to @thisisaaronland for merge. Property edits only (no-PIP).

**Comment by ndcartography** [comment]
👍

**Pull request merged**